ee/frontend/mobile-replay/wireframeStyle.ts

```diff
@@ -1,17 +1,33 @@
 import { MobileStyles, wireframe } from './mobile.types'
 
+function isNumber(candidate: unknown): candidate is number {
```

Comment:
Probably not in this PR but the codebase has many of those ` === 'number'` checks, maybe making this global/reusable and refactoring would make sense.

Reply:
Same for `=== 'string'` :D

Reply:
Yeah, was thinking this as I pulled them to one side.

We don't care so much in this codebase, but type checking into one place in posthog-js reduced bundle by about 1%.